chrome_manifest_v3/app.js

```
@@ -617,339 +606,330 @@ Gmail2Trello.App.prototype.markdownify = function (
+  const field_max_k = fields.length;
 
-    while (
-        (field1 = fields[fieldCount]) &&
-        valid &&
-        field_max_k > fieldCount++
-    ) {
-        if ((valid = obj_ptr.hasOwnProperty(field1))) {
-            obj_ptr = obj_ptr[field1];
-        }
+  while ((field1 = fields[fieldCount]) && valid && field_max_k > fieldCount++) {
+    if ((valid = obj_ptr.hasOwnProperty(field1))) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)